### PR TITLE
Added Default to `bootstrap` cmd

### PR DESCRIPTION
**Pull request by RichardLitt**
Part of #2484.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: afe1063059712de029107d0335f6dfc3b49f6d5e
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

We ask for a few features in the commit message for Open Source licensing hygiene and commit message clarity.
`git commit --amend` can often help you quickly improve the commit message.
[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help in the long run.
Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

**Comment by whyrusleeping** [comment]
I don't think the changed logic there is correct... I beleive it was correct before

File: core/commands/bootstrap.go [hunk 1]
```diff
@@ -160,10 +160,9 @@ var bootstrapRemoveCmd = &cmds.Command{
 		}
 
 		var removed []config.BootstrapPeer
+		removed, err = bootstrapRemove(r, cfg, input)
```
Comment (Kubuxu) [inline core/commands/bootstrap.go]: Are you sure that is correct?

**Comment by RichardLitt** [comment]
This is why we code review.

**Pull request merged**